app/decorators/host_decorator.rb

```diff
@@ -9,7 +9,10 @@ def fileicon
 
   def quadicon
     icon = {
-      :top_left     => {:text => vms.size},
+      :top_left     => {
+        :text    => t = vms.size,
+        :tooltip => n_("%{number} Virtual Machine", "%{number} Virtual Machines", t) % {:number => t}
+      },
       :top_right    => QuadiconHelper.machine_state(normalized_state),
```

Comment:
For Hosts, state is not being displayed

![screencast from 2018-06-13 17-02-40](https://user-images.githubusercontent.com/1187051/41359896-c9488566-6f2b-11e8-9c52-0035920b7518.gif)